qutebrowser/browser/downloads.py

```diff
@@ -286,6 +285,13 @@ def _ask_overwrite_question(self):
                                     window=self._win_id)
         message_bridge.ask(q, blocking=False)
 
+    def _download_dir(self):
+        """Get the download directory to use."""
+        download_dir = config.get('storage', 'download-directory')
+        if download_dir is None:
+            download_dir = standarddir.get(QStandardPaths.DownloadLocation)
+        return download_dir + os.sep
```

Comment:
Acording to the documentation this should be no problem. But either way, doing it in `path_suggestion` makes more sense.